packages/docusaurus/src/server/plugins/__tests__/__fixtures__/site-with-plugin/myPlugin.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (context, options) {
+    return {
```

Comment:
we usually intend with 2 spaces, not sure why prettier does not complain about this